### ISSUE TITLE
ImportError: cannot import name 'phase_cross_correlation' from 'skimage.registration'

### ISSUE THREAD

**Issue by Shaunakde** [scikit-image/scikit-image#4982]
## Description
I recently updated `skimage` and noticed that `phase_cross_correlation` is not present in the `registration` module. 

## Way to reproduce
```
1. Install skimage and numpy from `conda-forge`
2. Run `shift, error, diffphase = phase_cross_correlation(imga_ext, imgb_ext, upsample_factor=4)`
```

Running `dir` on the module with `dir(skimage.registration)` confirms this:
```
['__builtins__',
 '__cached__',
 '__doc__',
 '__file__',
 '__loader__',
 '__name__',
 '__package__',
 '__path__',
 '__spec__',
 '_optical_flow',
 '_optical_flow_utils',
 'optical_flow_tvl1']
```

Is this expected behaviour? I can't seem to find a depreciation notice anywhere for the module.

## Version information
```python
# your output here
3.7.6 (default, Jan  8 2020, 13:42:34) 
[Clang 4.0.1 (tags/RELEASE_401/final)]
Darwin-19.6.0-x86_64-i386-64bit
scikit-image version: 0.16.2
numpy version: 1.19.2
```

**Comment by Shaunakde** [comment]
For some reason, by default conda was not installing the newest package. Manually choosing a 0.17x release solved the issue!

**Comment by grlee77** [comment]
No problems, @Shaunakde. The function is available in v0.16.x releases, but was under a different name at that time. If you need to support multiple versions, the following would first try importing from its new location (v0.17.0+), but fall back to the old one if it is not found

```Python
try:
    from skimage.registration import phase_cross_correlation
except ImportError:
    from skimage.feature import register_translation as phase_cross_correlation
```
